spec/controllers/line_items_controller_spec.rb

```diff
@@ -60,8 +61,8 @@
         context "without an order cycle" do
           it "denies deletion" do
             delete :destroy, params
-            expect(response.status).to eq 403
-            expect { item.reload }.to_not raise_error
+            expect(response.status).to eq 401
```

Comment:
shouldn't this be a `403`? `401` is used for unauthenticated requests. `403` means that even if you're logged in you cannot perform that action.